modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptClientCodegen.java

```diff
@@ -1577,6 +1577,19 @@ protected void addImport(CodegenModel m, String type) {
         }
     }
 
+    protected void addImport(Set<String> importsToBeAddedTo, String type) {
+        if (type == null) {
+            return;
+        }
+
```

Comment:
some unit tests would be nice in https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/java/org/openapitools/codegen/typescript/TypeScriptClientCodegenTest.java

Reply:
can you add tests that are directly related to your code change? i.e. tests that would fail without your change.

Reply:
Ok
I try to do something 
but maybe it would be an example of this in the petstore and test there
I ll think over it